macros/cross_db_utils/ceil.sql

```diff
@@ -0,0 +1,11 @@
+{% macro ceil(field) -%}
+  {{ return(adapter.dispatch('ceil', packages = dbt_utils._get_utils_namespaces())(fields)) }}
+{%- endmacro %}
+
+{% macro default__ceil(field) -%}
+    ceil({{ field }})
+{%- endmacro %}
+
+{% macro alternative_ceil(field) %}
```

Comment:
Which adapters need to use `ceiling` over `ceil`? We'll want to adjust this to something like:
```suggestion
{% macro redshift__ceil(field) %}
```

Reply:
it's spark specifically (so dbx, but I thought it took the default then the alternative then database specific ones? since there was only on alternative (redshift, bq, and snowflake are all ceil), I thought one catchall might make more sense? If you don't think so, I can move this PR to spark_utils.

Reply:
I guess in both cases we'd need the default in dbt-utils, and the override in spark utils. Since none of our core adapters use `ceiling`, I think we'd want the spark-specific version over in spark-utils